templates/.github/ISSUE_TEMPLATE.md

```diff
@@ -1,7 +1,35 @@
-<!--
-1. Check the version of package you are using. If it's not the newest version, update and try again (see changelog while updating!).
-2. If the issue is still there, write a minimal project showing the problem and expected output.
-3. Link to the project and mention Node version and OS in your report.
+> Briefly describe the issue you are experiencing (or the feature you want to see
+added to the plugin). Tell us what you were trying to do and what happened
+instead. Remember, this is _not_ a place to ask questions. For that, go to
+http://gitter.im/webpack/webpack or https://stackoverflow.com/questions/tagged/webpack
 
-**IMPORTANT! You should use [Stack Overflow](https://stackoverflow.com/) for support related questions.**
--->
+### `📝  Details`
+
+> Describe in more detail the problem you have been experiencing, if necessary.
+
+### `❌ Error (Logs|Stacks)`
+
+> Create a [gist](https://gist.github.com) which is a paste of your **full**
+logs, and link them here.
+
+> ⚠️ Do **not** paste your full logs here (or at least hide them by using a `<details></details>` block), as it will make this issue long and hard
+to read! If you are reporting a bug, **always** include logs!
+
+### `♻️ Reproduction (Code)`
+
+> :warning: Please remember that, with sample code; it's easier to reproduce a bug and much
+faster to fix it.
+
+> 🔗 Please refer to a simple code example.
+
+```bash
+$ git clone https://github.com/<user>/<sample>
+```
+
+### `🌐 Environment`
+
+> ℹ️ Please provide information about your current environment.
+
+|OS|node|npm/yarn|package|
+|:-:|:--:|:-:|:------:|
+|[name][version]|[version]|[version]|[version]|
```

Comment:
`|[OS name][OS version]|[Node version]|[npm/Yarn version]|[package version]|` will be easier to fill.

Reply:
Best solution - give a command to run. The user can paste the result.